pygmt/src/logo.py

```diff
@@ -53,14 +55,12 @@ def logo(
         box is drawn using :gmt-term:`MAP_FRAME_PEN`. To customize the box appearance,
         pass a :class:`pygmt.params.Box` object to control style, fill, pen, and other
         box properties.
-    style : str
-        [**l**\|\ **n**\|\ **u**].
+    style
         Control what is written beneath the map portion of the logo.
 
-        - **l** to plot the text label "The Generic Mapping Tools"
-          [Default]
-        - **n** to skip the label placement
-        - **u** to place the URL to the GMT site
+        - ``"standard"``: The text label "The Generic Mapping Tools".
+        - ``"url"``: The URL to the GMT website.
+        - ``"no_label"``: Skip the text label.
```

Comment:
Maybe keep the l-n-u order?
```suggestion
        - ``"standard"``: The text label "The Generic Mapping Tools".
        - ``"no_label"``: Skip the text label.
        - ``"url"``: The URL to the GMT website.
```